### DISCUSSION TITLE
Why are there two "cars" in the labels file?

### DISCUSSION THREAD

**Discussion by julianrinaldi** [blakeblackshear/frigate#505] [Ask A Question]
Just wondering.

**Comment by blakeblackshear** [comment]
I renamed "truck" to car since they are often confused and it generates endless events as it switches back and forth.

**Reply by renjithstalin** [reply]
Sorry, for responding to this old thread. Does this mean the object truck doesn't work? I have truck in my track section but it doesn't do anything, all trucks are identified as cars.

**Reply by NickM-27** [reply]
You would also need to update your label map, but again keep in mind it will likely not work well.

**Reply by renjithstalin** [reply]
> You would also need to update your label map, but again keep in mind it will likely not work well.

Thank You, I will give it a try.
